plugins/SitesManager/lang/en.json

```diff
@@ -158,6 +158,31 @@
         "SiteWithoutDataGoogleTagManagerFollowStep9": "Preview your changes by clicking the \"Preview\" button.",
         "SiteWithoutDataGoogleTagManagerFollowStep10": "Check if the Matomo tag is firing as expected.",
         "SiteWithoutDataGoogleTagManagerFollowStep11": "Publish your changes by clicking the \"Submit\" button and then the \"Publish\" button at the top right.",
-        "SiteWithoutDataGoogleTagManagerFollowStepCompleted": "%1$sCongratulations!%2$s You have successfully installed the Matomo Analytics tracking code via Google Tag Manager. To verify that hits are being tracked, visit your website and check that this data is visible in your Matomo instance."
+        "SiteWithoutDataGoogleTagManagerFollowStepCompleted": "%1$sCongratulations!%2$s You have successfully installed the Matomo Analytics tracking code via Google Tag Manager. To verify that hits are being tracked, visit your website and check that this data is visible in your Matomo instance.",
+        "WordpressDetected": "Wordpress was detected on your website. Did you know you can use the Matomo easy WordPress plugin to track the website data? %1$sLearn more in this guide%2$s.",
```

Comment:
The phrase `the Matomo easy WordPress plugin` sounds a little funny. I think that `the easy Matomo WordPress plugin` or maybe `you can easily use the Matomo plugin for WordPress`.

Reply:
Yep, this would be great to have reworded.
I would also suggest to ask the product team to review the whole content by providing them with a screenshot of the UI in the internal ticket.

Reply:
@snake14 Updated as suggested and @michalkleiner The text was provided by the product team only

Reply:
> Looks good. The main issue I saw was that the Matomo Url is partially unreadable: ![image](https://user-images.githubusercontent.com/11394099/235007038-8a3a3f21-b42e-42b8-8dfb-ca2b63ed5761.png)

@snake14 This looks specific to your setup works fine for me, even the UI screenshot seems to display it correctly